scripts/pipelines/azure-devops/templates/package/package-pipeline.cfg

```
@@ -45,6 +46,16 @@ function addPipelineVariables {
       done
     fi
 
+    # We set the path to the file that contains the version number to be used as docker version.
+    if test -z ${imageTagFile}
+    then
+      for i in $dockerImageTagFilePaths
+      do
+        echo $i | grep $language > /dev/null && imageTagFile=$(echo $i | cut -d':' -f2)
+        break
```

```suggestion
        echo $i | grep $language > /dev/null && imageTagFile=$(echo $i | cut -d':' -f2) && break
```
I noticed I made an error because the break was getting executed on the first path even if it was not the right one